integration_tests/src/main/python/parquet_test.py

```diff
@@ -27,14 +27,20 @@ def read_parquet_df(data_path):
 def read_parquet_sql(data_path):
     return lambda spark : spark.sql('select * from parquet.`{}`'.format(data_path))
 
+
+# LIMIT: We have to use DECIMAL64(precision>8) to test decimal reading, because we only support DECIMAL64 in current.
```

Comment:
Either we need to be able to support something that is read as DECIMAL32 or we cannot ship this.  Also because of performance issues we are shipping 0.3 with decimal disabled by default #1278. That is until we can get a simple cost based optimizer in place or we have enough support for decimal that it is generally worth it. As such if this is hard to fix I'm not sure we need to get this into the 0.3 release.

Reply:
Hi @revans2, I added a type conversion for DECIMAL32 columns who are read via `cudf.readParquet`. It seems to work, but I am not sure whether this is a good approach.